MarkBind/markbind
# Make copy code button more subtle

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Enhancement to an existing feature



More colors: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value

**Current button**
![image](https://user-images.githubusercontent.com/38814428/95484612-453f9680-09c3-11eb-95c7-37f507439553.png)
On hover:
![image](https://user-images.githubusercontent.com/38814428/95484651-4f619500-09c3-11eb-8000-d4f2da5dea0e.png)

**Current alternative buttons**
![image](https://user-images.githubusercontent.com/38814428/95472147-44a00380-09b5-11eb-9678-b3ff02e5165c.png)



**What is the rationale for this request?**
Addresses #1278 by changing the look-and-feel of the code copy button.

**What changes did you make? (Give an overview)**
Update the CSS for code block buttons, remove unused css, update the icon used for the copy button. Updated user guide to reflect the changed copy code block css.

**Provide some example code that this change will affect:**


```js

```

**Is there anything you'd like reviewers to focus on?**


**Testing instructions:**
1. Serve any copy of markbind with code blocks enabled.
2. Verify that the button works and looks as intended.

**Proposed commit message: (wrap lines at 72 characters)**
Make copy code button more subtle
